### ISSUE TITLE
Ran into trouble with CouchDB 2.0 preview

### ISSUE THREAD

**Issue by jmarca** [pouchdb/add-cors-to-couchdb#15]
(Oops hit enter not tab.)

I didn't see this issue already opened, so I thought I'd mention that this doesn't seem to work with CouchDB master (the 2.0 preview, pulled & compiled Oct 6, 2015)

Below is the output.  I tried both with the haproxy front end on port 5985, and with one of the server nodes on port 15984, and again without a password.

```
james@emma couchdb[master]$ add-cors-to-couchdb http://root:97XbZYWV@127.0.0.1:5985
[Error: got a 404 code]
james@emma couchdb[master]$ add-cors-to-couchdb http://root:97XbZYWV@127.0.0.1:15984
[Error: got a 404 code]
james@emma couchdb[master]$ add-cors-to-couchdb http://127.0.0.1:15984
[Error: got a 302 code]
```

(Yeah, that's the auto-generated root password...it is okay because it generates a new one every time.)

Had no problems at all with this on 1.6.x, of course.

**Comment by nolanlawson** [comment]
Yeah I noticed this as well. It doesn't work for CouchDB 2.0 yet. @garrensmith @willholley interested in looking into this? :D

**Comment by garrensmith** [comment]
The url for changing a config has changed. Its now

`http://127.0.0.1:15984/_node/<fqdn>/_config`

Where `<fqdn>` is the node name. So if you building off master and running something like

`./dev/run -n 1` 

Then fqdn is `node1@127.0.0.1`.

I've been working on [nmo](https://github.com/apache/couchdb-nmo) and it might be the better way to do this. I've just added in support for changing the [config](https://github.com/apache/couchdb-nmo/blob/master/doc/cli/nmo-couch-config.md)

**Comment by jmarca** [comment]
Ah nmo does seem useful. I hadn't noticed that yet. Far better than the suggestion I saw in Fauxton to use "Chef, Ansible, Puppet, or Salt" to configure, as I haven't ever used any of those.  I'll try out your improvements this weekend if I have time.

**Comment by janl** [comment]
More detailed description for how to address this:

https://github.com/klaemo/docker-couchdb/issues/42#issuecomment-169610897

cc @klaemo @cquillen2003

**Comment by nolanlawson** [comment]
fixed in #15
